tests/Feature/Controllers/ProjectsControllerTest.php

```
@@ -0,0 +1,226 @@
+        $response = $this->get(route('projects.index'));
+
+        /** Assert */
+        $response->assertOk();
+        $response->assertViewIs('projects::projects_index');
+        $response->assertViewHas('projects');
+
+        /** Verify project is in the list */
+        $projects = $response->viewData('projects');
+        $projectIds = $projects->pluck('project_id')->toArray();
+        $this->assertContains($project->project_id, $projectIds);
+    }
```

PHPMD (2.15.0)

The method it_displays_list_of_projects is not named in camelCase. (undefined)

(CamelCaseMethodName)